src/slashcommands/ISlashCommand.ts

```diff
@@ -0,0 +1,6 @@
+// just guessing
+export interface ISlashCommand {
+    command: string;
+    pattern: string;
+    function: () => boolean;
```

Comment:
Actually I did mean for it to be specific to the slash commands, however with you asking whether it was referring to the generic `execute` methods made me realize that commands should be no different than how we are handling the other `execute` methods.

And yes, that was completely understandable. :1st_place_medal: